chartiq/development/js/standard.js

```diff
@@ -7566,7 +7566,7 @@ CIQ.ChartEngine.prototype.findHighlights = function (isTap, clearOnly) {
 		this.setMeasure();
 	}
 
-	if (highlightedDraggable && !myPanel.noDrag) {
+	if (highlightedDraggable && !myPanel?.noDrag) {
```

Comment:
This condition will be truthy if `myPanel` is undefined too. We should check if `myPanel` is defined first.

Suggested Change:
```
if (highlightedDraggable && myPanel && !myPanel.noDrag) {
```

Reply:
Done, Thank you)